recipe/meta.yaml

```
@@ -78,61 +100,109 @@ outputs:
         - if not exist "%LIBRARY_BIN%"\\llvm-nm.exe exit 1              # [win]
         - llvm-nm.exe --help                                            # [win]
 
+        # ensure we've correctly inserted %VSINSTALLDIR% into the CMake metadata for LLVM;
+        # we're looking for: `INTERFACE_LINK_LIBRARIES "$ENV{VSINSTALLDIR}/DIA SDK/lib/amd64/diaguids.lib;[...]`instead
+        - rg -e "INTERFACE_LINK_LIBRARIES\s\"\$ENV\{VSINSTALLDIR\}[/\w\s]+/diaguids\.lib" %LIBRARY_LIB%\cmake\llvm\LLVMExports.cmake  # [win]
+
   # Contains the shared libraries. To make different LLVM libraries co-installable
   # soversion is appended to the package name.
   - name: libllvm{{ major_ver }}
     script: install_llvm.sh  # [not win]
     build:
+      activate_in_script: true
       skip: true  # [win and vc<14]
-      run_exports:   # [not win]
-        - {{ pin_subpackage("libllvm" + major_ver, max_pin="x.x") }}  # [not win]
+      run_exports:                                                    # [not win]
+        - {{ pin_subpackage("libllvm" ~ major_ver, max_pin="x.x") }}  # [not win]
       missing_dso_whitelist:  # [s390x]
         - $RPATH/ld64.so.1    # [s390x]
     requirements:
       build:
         - {{ compiler('cxx') }}
-        - ninja                  # [not win]
-        - cmake                  # [not win]
-        - python >=3             # [not win]
+        - ninja                    # [not win]
+        - cmake                    # [not win]
+        - python >=3               # [not win]
       host:
         - libcxx {{ cxx_compiler_version }}  # [osx]
-        - libffi {{ libffi }}  # [unix]
-        - libxml2 {{ libxml2 }}  # [win]
-        - zlib {{ zlib }}
-        - zstd {{ zstd }}
+        - libxml2   # [unix]
```

```suggestion
        - libxml2  {{ libxml2 }}
```
Probably still want to version it, right? Also, this output seems to use `build.bat` which has `-DLLVM_ENABLE_LIBXML2=FORCE_ON ^` so needed for all platforms.